contracts/allocators/LUSDAllocator.sol

```diff
@@ -205,7 +216,9 @@ contract LUSDAllocator is Ownable {
         address _stabilityPool,
         address _lqtyStaking,
         address _frontEndAddress,
-        address _wethAddress
+        address _wethAddress,
+        address _hopTokenAddress,
+        address _uniswapV3Router
     ) {
         setTreasury(_treasury);
```

Comment:
Would use Olympus authority. Would need to inherit from Olympus Access Control.

`constructor(IOlympusAuthority _authority) OlympusAccessControlled(_authority) {}`

Reply:
So instead of onlyOwner can use use onlyGuardian or onlyGovernor

Reply:
Done!